support.sh

```diff
@@ -102,3 +113,17 @@ echo -e "\n\n==SUMMARY=="
 echo -e "\t Processed $networks networks"
 echo -e "\t IP overlap found: $ip_overlap"
 echo -e "\t Processed $containers containers"
+
+if [ "true" == ${SSD} ] ; then
```

Comment:
not sure if the SSD has some output, but I would like the SUMMARY to be the last thing to be printed at the end

Reply:
${SSD} is just something getting passed from another script to enable the ssd code. If I've understood your question correctly?

Reply:
I put the comment on the top of the block but was referring to the underlying code. I would like to keep the summary as the last output produced from the script, so you need to move your change on top